test/components/formcomponent.test.js

```diff
@@ -105,6 +116,16 @@ lab.experiment('FormComponent', () => {
         const viewModel = context.formComponent.getViewModel({}, formData)
         expect(viewModel.errorMessage).to.not.exist()
       })
+      lab.test('label is page heading', ({ context }) => {
+        const viewModel = context.formComponent.getViewModel({}, formData)
+        expect(viewModel.label).to.exist()
+        expect(viewModel.label.isPageHeading).to.equal(true)
+      })
+      lab.test('label is large', ({ context }) => {
+        const viewModel = context.formComponent.getViewModel({}, formData)
+        expect(viewModel.label).to.exist()
+        expect(viewModel.label.classes).to.equal('govuk-label--l')
+      })
```

Comment:
This functionality already exists and the tests are in the `page.test.js` test suite.

Reply:
thanks, I'll get the tests and extra code removed.

Reply:
thanks, @bensagar-ea ready for re-review - I've improved the example register code, and removed the duplicate code and tests.